### PR TITLE
[Angular] Add input tests

### PR DESCRIPTION
**Pull request by dhoanglong91epam**

### CODE REVIEW

**Comment by mf-code** [comment]
[5624] - 5JdCTSfU

**Comment by mf-code** [comment]
[5624] - HAMd00pj

**Comment by mf-code** [comment]
[5624] - 5iLoSF06

**Comment by mf-code** [comment]
[5624] - hamULWi6

**Comment by mf-code** [comment]
[5624] - Allure report on Netlify: https://5ee88fd28cf279e5399fdc8f--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java [hunk 1]
```diff
@@ -51,4 +52,52 @@
     public static Checkbox configurableResultCheckbox;
 
     public static SlideToggleSection slideToggleSection;
+
+    @UI("#mat-input-12")
+    public static TextField foodBasicInput;
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java:57]: Please create a separate sections for all inputs. We should not put all the small elements to the same AngularPage, this way we will end up with huge amount of elements defined in one class.

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java [hunk 2]
```diff
@@ -51,4 +52,52 @@
...
+    public static TextField messageHintInput;
+
+    @UI(".ng-tns-c94-53 mat-hint:first-of-type")
+    public static Text messageHint;
+
+    @UI(".ng-tns-c94-53 mat-hint:last-of-type")
+    public static Text messageCounterHint;
+
+    @UI(".ng-tns-c94-54 .mat-form-field-prefix")
+    public static UIElement prefixInput;
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java:96]: If in jdi-light-html there is no suitable element, then please add this element to jdi-light-angular and use it intead of UIElement.

**Comment by mf-code** [comment]
[5635] - FDgraaBl

**Comment by mf-code** [comment]
[5635] - Tjr8aFVV

**Comment by mf-code** [comment]
[5635] - fj4czyVK

**Comment by mf-code** [comment]
[5635] - 6DH1UUNs

**Comment by mf-code** [comment]
[5635] - Allure report on Netlify: https://5ee9cab48732fad80c28d77a--musing-swartz-6fc060.netlify.app

**Pull request closed without merging**